docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -17,9 +17,10 @@ This article describes what is new in .NET Core 3.0 (preview 2). One of the bigg
 
 [Download and get started with .NET Core 3 Preview 2](https://aka.ms/netcore3download) right now on Windows, Mac and Linux. You can see complete details of the release in the [.NET Core 3 Preview 2 release notes](https://aka.ms/netcore3releasenotes).
```

Comment:
@Thraka it would be good to be consistent about how we refer to the release. I see .NET Core 3 and .NET Core 3,0 being used. Which one is the right one?